gradio/routes.py

```diff
@@ -479,6 +479,7 @@ async def join_queue(
         async def get_queue_status():
             return app.get_blocks()._queue.get_estimation()
 
+        @app.on_event("startup")
```

Comment:
So that the queue can start with calling launch. I checked and there's no problem in calling this route twice on `startup` and during `launch`